README.md

```diff
@@ -144,3 +144,122 @@ assigned to the "AppSecurityGroup" so they can talk to the database as
 well as the load balancer, and receive SSH traffic from the bastion host.
 
 
+### Deploy
+
+New services are not always available in all AWS Regions when they are released. Please consult the
+[Region Table](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/) for
+more information.
+
+Deploy | Region Name | Region | VPC | Bastion | DB | Fargate | Elastic Beanstalk
```

Comment:
This is very hard to parse for me, and I pretend to know what I'm doing. I think it's going to be very confusing for folks to narrow down to the right option with this mega-table.

I'd recommend at a minimum breaking this up into paths. Looking at the rows, you're working to represent at most five different paths depending upon Region:

- Basic Infrastructure (VPC)
- Basic Infrastructure (VPC) + Bastion
- Basic Infrastructure (VPC) + Bastion + App (Fargate)
- Basic Infrastructure (VPC) + Bastion + App (Fargate) + DB 
- Basic Infrastructure (VPC) + Bastion + App (Elastic Beanstalk) + DB 

Breaking these into five tables would it easier for a user to select what they need. They can first figure out which table (i.e. what infrastructure they'd like to create) and then from that table select a region. It also really calls out the composability of what you're building here.

Reply:
I have an updated README, really a total rewrite, coming in a separate PR--I think we need to:
- Review this PR, give it a thumbs-up (when appropriate), but hold off on merging to master
- Do same for README PR
- Merge them back-to-back, since @rnzsgh's PR will obsolete the current README, and the new README references the new material.

@koroshan has a CF template for billing alerts forthcoming. I'm going to remove the description of that from the README and add it back once Roshan's work is merged.

Reply:
I added a SIM ticket and assigned to @john. I agree with breaking things down by stack, but it could turn into a series of huge tables. Does it make sense to create separate doc files, with tables for each cluster of functionality? I need to get it up the way it is now, because I have people waiting for me to send them the link.